Yousei.Connectors/Http/HttpRequest.cs

```diff
@@ -61,7 +61,7 @@ public static async Task<HttpRequest> FromRequest(HttpListenerRequest request)
             Dictionary<string, string> ToDictionary(NameValueCollection nameValueCollection)
                 => nameValueCollection
                     .Cast<string>()
-                    .ToDictionary(o => o, o => nameValueCollection[o]);
+                    .ToDictionary(o => o, o => nameValueCollection[o] ?? string.Empty);
```

Comment:
How would the Dictionary look like here?

Reply:
Do you mean the type or the content?

Reply:
The content

Reply:
This dictionary consists of HTTP header as keys and their respective values.